packages/manager/apps/cloud/client/app/dbaas/logs/detail/home/logs-home.controller.js

```diff
@@ -43,7 +43,9 @@ class LogsHomeCtrl {
             if (service.state === this.LogsConstants.SERVICE_STATE_TO_CONFIG) {
               this.goToAccountSetupPage();
             } else {
-              this.dataUsageGraphData = this.LogsConstants.DATA_USAGE_GRAPH_CONFIGURATION;
+              this.streamUsageGraphData = this.newChart('y-axis-1');
+              this.archiveUsageGraphData = this.newChart('y-axis-2');
+              this.indiceUsageGraphData = this.newChart('y-axis-3');
```

Comment:
Maybe we could use better chart ids that reflect what the chart represents ? 
```suggestion
              this.streamUsageGraphData = this.newChart('stream-axis');
              this.archiveUsageGraphData = this.newChart('archive-axis');
              this.indiceUsageGraphData = this.newChart('indice-axis');
```

Reply:
Fixed